independent-publisher-connectors/Finnish BIS/apiProperties.json

```diff
@@ -0,0 +1,9 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#da3b01",
+    "capabilities": [],
+    "publisher": "Timo Pertilä",

```

Comment:
@tpertila Unicodes - The connector's swagger still have unicodes which may not be supported in all the Power Automate page, Can you please replace all the unicodes with respective characters in the English language.

Reply:
@tpertila Can you please provide the publisher value in English language ?